priv/repo/seeds.exs

```diff
@@ -18,6 +18,7 @@ Repo.transaction(fn ->
     "external_id" => tenant_name,
     "jwt_secret" =>
       System.get_env("API_JWT_SECRET", "super-secret-jwt-token-with-at-least-32-characters-long"),
+    "max_channels_per_client" => String.to_integer(System.get_env("MAX_CHANNELS_PER_CLIENT") || "100"),
```

Comment:
always use the new value on new clients, default back to system default of 100

Reply:
here we could the same as I mentioned above, move this checks to `runtime.exs` and then do a `Application.get_env(:realtime, :max_channels_per_client)`